tests/features/bootstrap/KataContext.php

```diff
@@ -119,19 +107,28 @@ public function iEvaluateTheKata($kataName)
 
     private function runCommand($command, array $args, $silenceOutput = false)
     {
-        $input = new ArrayInput(
-            array_merge(
-                array(
-                    'command' => $command,
-                ),
-                $args
-            )
-        );
-        $this->application->setAutoExit(false);
-        $output = $this->output;
-        if ($silenceOutput) {
-            $output = new NullOutput();
+        $phpBinary = new PhpExecutableFinder();
+
+        $processBuilder = new ProcessBuilder([
+            'phpkata.php',
+            $command
+        ] + $args);
```

Comment:
yes I know you won't like this "version" of an `array_merge`.. it keeps the ones in the first array if there's conflicts. Don't think it's relevant here I was just testing the command.